File: Architecture.Core/Entities/ProfDelegationInfo.cs
using System;
using Architecture.Core.Entities.Core;

namespace Architecture.Core.Entities
{
    public class ProfDelegationInfo : Auditable
    {
        public int DelegationInfoId { get; set; }
        public int ProfileId { get; set; }
        public virtual ProfBasicInfo ProfBasicInfo { get; set; }
        public string Name { get; set; }
        public string SurName { get; set; }
        public DateTime DateOfBirth { get; set; }
        public string TaxCode { get; set; }
        public string RefNo { get; set; }
        public string Purpose { get; set; }
        public DateTime DocumentIssueDate { get; set; }
        public DateTime ExpiryDate { get; set; }
        public string IssuedBy { get; set; }
        public string Status { get; set; }

    }
}

File: Architecture.Web/ClientApp/src/app/Shared/Entity/ClientProfile/profDelegationInfo.ts
import { Data } from '@angular/router';
import { Auditable } from '../Common/auditable';

export class profDelegationInfo extends Auditable {
    public  DelegationInfoId: number;
    public profileId: number;
    //public virtual ProfBasicInfo ProfBasicInfo { get; set; }
    
    public name: string;

    public surName: string;

    public dateOfBirth: Date;

    public taxCode: string;

    public refNo: string;

    public purpose: string;

    public documentIssueDate: Date;

    public expiryDate: Date;

    public issuedBy: string;

    public status: string;
        
}


